app/src/main/kotlin/com/wire/android/ui/theme/WireColorScheme.kt

```diff
@@ -142,7 +142,7 @@ private val LightWireColorScheme = WireColorScheme(
     background = WireColorPalette.Gray20, onBackground = Color.Black,
     backgroundVariant = WireColorPalette.Gray10, onBackgroundVariant = Color.Black,
     surface = Color.White, onSurface = Color.Black,
-    surfaceVariant = Color.White, onSurfaceVariant = Color.Black,
+    surfaceVariant = Color.White , onSurfaceVariant = Color.Black,
```

Comment:
question: the description of this PR says that these fields were not correct according to the designs but here they are not updated at all and still have the same values as before this PR - shouldn't that be changed then? 🤔

Reply:
You are right. In the end we didn't have to modify anything on the color scheme, but refer to the correct color schemes from affected composables. I will rephrase the PR description accordingly too.